osgi-dependencies/proxy/src/main/java/com/cognifide/aet/proxy/ProxyServerProvider.java

```diff
@@ -44,19 +44,20 @@
           unbind = "unbindProxyManager")
   private final Map<String, ProxyManager> collectorManagers = Maps.newConcurrentMap();
 
-  public ProxyServerWrapper createProxy(String useProxy, int port) throws ProxyException {
-    String proxyType = useProxy;
-    if ("true".equals(useProxy)) {
+  public ProxyServerWrapper createProxy(String usedProxy) throws ProxyException {
+    String proxyType = usedProxy;
+    if ("true".equals(usedProxy)) {
```

Comment:
This is clearly some configuration legacy. Before Browsermob, AET used another proxy (embedded) and then `useProxy` configuration could have 2 values: `true` or `false`. When the browsermob came, because of a backward compatibility, configuration was extended to `rest` and `embedded`.
In time, embedded proxy option was removed from the system as it didn't work well.
Thank you @wiiitek for noticing this.
I think we have 2 paths now:
- lets leave `useProxy` possible values as `true` and `rest` which are equivalent (`true` indicates that the *default* proxy will be used).
- there will be only named type of a proxy (remove `true` as a possible value) after deprecating it first for e.g. 2 minor versions?

What do you think?